db/migrate/20231024214652_create_posts.rb

```diff
@@ -11,4 +11,5 @@ def change
     end
     add_foreign_key :posts, :users, column: :author_id
   end
-end
\ No newline at end of file
+end
+Post.create(title: 'post3', author_id: 1, text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas a libero at ex fermentum auctor. Donec ac semper justo. Sed tincidunt quam vel massa consectetur.', comments_counter: 0, likes_counter: 0)
```

Comment:
- [ ] Please move this line of code into the `seeds.rb` file. 🙏 
_See the first comment for more details._